plugins/workspace-plugin/src/theia-commands.ts

```diff
@@ -207,46 +178,57 @@ export class TheiaGitCloneCommand implements TheiaImportCommand {
         return Promise.reject(new Error(message));
       }
 
-      break;
+      // pause will be removed after debugging this method
+      await this.pause(100);
```

Comment:
It looks like this PR should be aligned with master. I just remember I've removed this delay, but it's still present here.

Reply:
fixed